barretenberg/cpp/src/barretenberg/vm2/common/tagged_value.hpp

```diff
@@ -12,13 +12,13 @@
 namespace bb::avm2 {
 
 enum class ValueTag {
-    FF,
-    U1,
-    U8,
-    U16,
-    U32,
-    U64,
-    U128,
+    FF = 0,
```

Comment:
Will not block but you _MIGHT_ want to define same values in constants.nr and use them here and PIL etc. Like `TAG_ENUM_FF = 0`...

Reply:
Fortunately we already have these in the form for MEM_TAG_X